### PR TITLE
Split commands

### PR DESCRIPTION
**Pull request by Lazersmoke**
Untested because namelayer borked my local server :/

### CODE REVIEW

File: src/main/java/com/github/maxopoly/MemeMana/command/CmdManaGive.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import java.util.LinkedList;
+import java.util.function.IntFunction;
+
+public class CmdManaGive extends PlayerCommand {
+	public CmdManaGive(String name) {
+		super(name);
+		setIdentifier("managive");
+		setDescription("Give a player some mana");
+		setUsage("/managive Player Amount");
+		setArguments(2,2);
```
Comment (Maxopoly) [inline src/main/java/com/github/maxopoly/MemeMana/command/CmdManaGive.java]: Add a short alias for all the commands that people can easily use like mmg (meme mana give)

File: src/main/java/com/github/maxopoly/MemeMana/command/CmdManaGive.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+			msg("<i>%s <b>is not a valid amount of mana",args[2]);
+			return false;
+		}
+		MemeManaPlugin.getInstance().getManaManager().addMana(owner,giveAmount);
+		msg("<g>Gave <c>%s <i>%d<g> mana",args[1],giveAmount);
+		return true;
+	}
+
+	public List <String> tabComplete(CommandSender sender, String [] args) {
+		return new LinkedList <String> (); //empty list
```
Comment (Maxopoly) [inline src/main/java/com/github/maxopoly/MemeMana/command/CmdManaGive.java]: Why not complete to player names here?

File: src/main/java/com/github/maxopoly/MemeMana/command/CmdManaShow.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+	public boolean execute(CommandSender sender, String [] args) {
+		if (!(sender instanceof Player)) {
+			msg("Can't show your own mana from console.");
+			return true;
+		}
+		MemeManaPlayerOwner owner = MemeManaPlayerOwner.fromPlayer((Player)sender);
+		MemeManaPouch pouch = MemeManaPlugin.getInstance().getManaManager().getPouch(owner);
+		double manaAvailable = pouch.getContent();
+		msg("<i>You have<g> %s<i> mana",String.valueOf(manaAvailable));
```
Comment (Maxopoly) [inline src/main/java/com/github/maxopoly/MemeMana/command/CmdManaShow.java]: Cut off decimal at a certain point by using something like https://docs.oracle.com/javase/7/docs/api/java/text/DecimalFormat.html

File: src/main/java/com/github/maxopoly/MemeMana/command/CmdManaGive.java (hunk 1 above)
Comment (Lazersmoke) [reply src/main/java/com/github/maxopoly/MemeMana/command/CmdManaGive.java]: How do I do this? Just another `setIdentifier("mmg");`?

**Pull request merged**